### ISSUE TITLE
Failed Connection to NDI Aurora

### ISSUE THREAD

**Issue by asgardee** [PlusToolkit/PlusLib#584]
Hello,

I recently tried launching a server for my NDI Aurora device and the connection failed. I have attached a copy of my .xml file (in .txt format as .xml is not supported on git) and my NDI Trace information. I have tried following the advice from ticket #347 but was unable to fix my problem. I have confirmed that my port number is COM3 and I have 3x 6DOF sensors that are connected to my device (that work in the NDI Track application).

Any help on this issue would be greatly appreciated!

[Aurora_Tracker_xml.txt]
(https://github.com/PlusToolkit/PlusLib/files/3494914/Aurora_Tracker_xml.txt)

[NDI_Trace.txt](https://github.com/PlusToolkit/PlusLib/files/3494919/NDI_Trace.txt)

**Comment by Sunderlandkyl** [comment]
Can you attach a log file?

**Comment by asgardee** [comment]
Hey Kyle, I'm currently away from the lab but I will upload a log file in 3-4 hours (when I return). Thank you for following up and I apologize for the time delay!

**Comment by asgardee** [comment]
Hello Kyle, I have attached the PlusLog file to this message. For reference when I attempt to launch the server my Aurora device beeps twice (similar to a successful launch) but then fails to initialize. Thank again for the help.

[081419_114516_PlusLog.txt](https://github.com/PlusToolkit/PlusLib/files/3502599/081419_114516_PlusLog.txt)

**Comment by asgardee** [comment]
Hello,

After some extra troubleshooting, I was able to find out the issue. The sensors that we had bought from NDI were not prefabricated, meaning that we had to load a virtual SROM to the device in the tracking software (NDI Track). This virtual SROM is saved to the NDI Track software, but in order to communicate with the devices outside of that software, you must include ROM files. The ROM files for each device can be found on the [NDI support site](https://support.ndigital.com/downloads.php), and you must include the path to the ROM files in your xml definition. I have uploaded the updated xml file to this post, with the key difference being in the 'DataSource' definitions.

I hope this helps anyone else who may run into this problem!

[Aurora_Tracker_xml.txt](https://github.com/PlusToolkit/PlusLib/files/3503108/Aurora_Tracker_xml.txt)
